raven_word_count.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
This is totally legit, and I like it.  For really short jobs, it's just as efficient to put them in the makefile directly too, which helps keep the root directory from cluttering up with small scripts, but it's a give and take.